assets/app/view/game/buy_special.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+
+module View
```

Comment:
so what is this used for?

Reply:
To purchase a coal marker. I made it general, so that other games that have game specific purchases, can reuse it. 1856 and the tunnel/bridge access, OE with port access, pullmans, and more.